From: Benjamin Drung <benjamin.drung@canonical.com>
Date: Tue, 8 Oct 2024 16:40:48 +0200
Subject: Move UNIX System V zones back from backzone to backwards

Commit a0b09c023008 ("Mark CET, CST6CDT etc. as obsolescent") and
782d082623aa ("Make EET, MET and WET links") replaced the obsolete UNIX
System V zones by symlinks to geographical names. This can be a breaking
change in case users wanted to use a zone with DST changes.

Move the previous UNIX System V zones back from the `backzone` to the
`backwards` file.
---
 backward | 31 ++++++++++++++++++++-----------
 backzone | 21 ---------------------
 2 files changed, 20 insertions(+), 32 deletions(-)

diff --git a/backward b/backward
index 1e5de57..3efd238 100644
--- a/backward
+++ b/backward
@@ -45,8 +45,6 @@ Link	America/Rio_Branco	Brazil/Acre
 Link	America/Noronha		Brazil/DeNoronha
 Link	America/Sao_Paulo	Brazil/East
 Link	America/Manaus		Brazil/West
-Link	Europe/Brussels		CET
-Link	America/Chicago		CST6CDT
 Link	America/Halifax		Canada/Atlantic
 Link	America/Winnipeg	Canada/Central
 # This line is commented out, as the name exceeded the 14-character limit
@@ -61,9 +59,6 @@ Link	America/Whitehorse	Canada/Yukon
 Link	America/Santiago	Chile/Continental
 Link	Pacific/Easter		Chile/EasterIsland
 Link	America/Havana		Cuba
-Link	Europe/Athens		EET
-Link	America/Panama		EST
-Link	America/New_York	EST5EDT
 Link	Africa/Cairo		Egypt
 Link	Europe/Dublin		Eire
 # Vanguard section, for most .zi parsers.
@@ -102,9 +97,6 @@ Link	America/Jamaica		Jamaica
 Link	Asia/Tokyo		Japan
 Link	Pacific/Kwajalein	Kwajalein
 Link	Africa/Tripoli		Libya
-Link	Europe/Brussels		MET
-Link	America/Phoenix		MST
-Link	America/Denver		MST7MDT
 Link	America/Tijuana		Mexico/BajaNorte
 Link	America/Mazatlan	Mexico/BajaSur
 Link	America/Mexico_City	Mexico/General
@@ -299,7 +291,6 @@ Link	Europe/Kyiv		Europe/Zaporozhye
 Link	Pacific/Kanton		Pacific/Enderbury
 Link	Pacific/Honolulu	Pacific/Johnston
 Link	Pacific/Port_Moresby	Pacific/Yap
-Link	Europe/Lisbon		WET
 
 
 # Alternate names for the same location
@@ -325,7 +316,25 @@ Link	Europe/Kyiv		Europe/Kiev
 # Classically, Cyprus is in Asia; e.g. see Herodotus, Histories, I.72.
 # However, for various reasons many users expect to find it under Europe.
 Link	Asia/Nicosia		Europe/Nicosia
-Link	Pacific/Honolulu	HST
-Link	America/Los_Angeles	PST8PDT
 Link	Pacific/Guadalcanal	Pacific/Ponape	#= Pacific/Pohnpei
 Link	Pacific/Port_Moresby	Pacific/Truk	#= Pacific/Chuuk
+
+# From Paul Eggert (2024-05-22):
+# The following zones pretend that standard time extends backward
+# indefinitely into the past, and so are ahistorical.
+# In current TZDB these entries are links to geographical locations
+# that agree with the ahistorical zones since 1970.
+# These are in numeric rather than alphabetic order.
+
+# Zone	NAME		STDOFF	RULES	FORMAT	[UNTIL]
+Zone	HST		-10:00	-	HST
+Zone	PST8PDT		 -8:00	US	P%sT
+Zone	MST		 -7:00	-	MST
+Zone	MST7MDT		 -7:00	US	M%sT
+Zone	CST6CDT		 -6:00	US	C%sT
+Zone	EST		 -5:00	-	EST
+Zone	EST5EDT		 -5:00	US	E%sT
+Zone	WET		  0:00	EU	WE%sT
+Zone	CET		  1:00	C-Eur	CE%sT
+Zone	MET		  1:00	C-Eur	ME%sT
+Zone	EET		  2:00	EU	EE%sT
diff --git a/backzone b/backzone
index dd7a867..f73942b 100644
--- a/backzone
+++ b/backzone
@@ -1827,27 +1827,6 @@ Zone	Pacific/Wake	11:06:28 -	LMT	1901
 Zone	Pacific/Wallis	12:15:20 -	LMT	1901
 			12:00	-	+12
 
-
-# From Paul Eggert (2024-05-22):
-# The following zones pretend that standard time extends backward
-# indefinitely into the past, and so are ahistorical.
-# In current TZDB these entries are links to geographical locations
-# that agree with the ahistorical zones since 1970.
-# These are in numeric rather than alphabetic order.
-
-# Zone	NAME		STDOFF	RULES	FORMAT	[UNTIL]
-Zone	HST		-10:00	-	HST
-Zone	PST8PDT		 -8:00	US	P%sT
-Zone	MST		 -7:00	-	MST
-Zone	MST7MDT		 -7:00	US	M%sT
-Zone	CST6CDT		 -6:00	US	C%sT
-Zone	EST		 -5:00	-	EST
-Zone	EST5EDT		 -5:00	US	E%sT
-Zone	WET		  0:00	EU	WE%sT
-Zone	CET		  1:00	C-Eur	CE%sT
-Zone	MET		  1:00	C-Eur	ME%sT
-Zone	EET		  2:00	EU	EE%sT
-
 # Local Variables:
 # coding: utf-8
 # End:
